zcash_proofs/Cargo.toml

```diff
@@ -17,7 +17,7 @@ categories = ["cryptography::cryptocurrencies"]
 all-features = true
 
 [dependencies]
-zcash_primitives = { version = "0.10", path = "../zcash_primitives" }
+zcash_primitives = { version = "0.10", path = "../zcash_primitives", default-features = false }
```

Comment:
Add `zcash_primitives/multicore` to the `multicore` feature flag below. They are logically tied together, and if we ever move either Orchard proving into `zcash_proofs` or Sapling proving out of it, we'd be doing the same for `bellman` and `halo2_proofs` directly.